Fix/doc 4253 (#51)
exoplatform/exo-documentation: docs/Configuration.rst

    -  :ref:`WebDAV configuration <Configuration.WebDAV>`
       Configuration of the WebDAV service.
       

>     -  :ref:`Secure access to WebDAV urls of JCR folders <Configuration.SecureAccessWebdavJCR>`

Reviewer: I prefer the following title : 
```suggestion
    -  :ref:`Secure the listing of the contents of JCR folders through Webdav <Configuration.SecureAccessWebdavJCR>`
```

>        How to secure the access to WebDAV urls of JCR folders.

Reviewer: ```suggestion
       How to secure the listing of the contents of folders through Webdav URLs.
```

|                                     |cached.                               |
+-------------------------------------+--------------------------------------+

.. _Configuration.SecureAccessWebdavJCR:

==================================================================
Secure the listing of the contents of JCR folders through Webdav
==================================================================

For security reasons, it is important to an administrator to secure the 
access to WebDAV urls of JCR folders.

A parameter ``exo.webdav.folder.listing.paths.allowed.regex`` is introduced 

> in eXo Platform allowing to define the set of WebDAV urls to JCR folders 

Reviewer: ```suggestion
in eXo Platform allowing listing the contents of JCR folders when they are accessed with Webdav 
```

> to be listed.

Reviewer: to be removed

> to be listed.

Reviewer: ```suggestion

```

> This parameter defines a regular expression of the allowed WebDav urls 

Reviewer: ```suggestion
This parameter defines a regular expression for the folders that could be listed  
```

> of JCR folders:

Reviewer: to be removed

Author: i changed the sentence

> of JCR folders:

Reviewer: ```suggestion
```

::

		exo.webdav.folder.listing.paths.allowed.regex=(collaboration:/Users/(.*)/(.*)/(.*)/(.*))|(collaboration:/Groups/(.*))|(collaboration:/sites/(.*))|(portal-system:/production/app:gadgets/(.*))

The above example allows the listing access to theses folders: users folders, 
groups folders, sites folders and gadgets folders.

.. note:: The value of the parameter ``exo.webdav.folder.listing.paths.allowed.regex``

>           should have this form: **wokspace_Name:/regex**.

Reviewer: ```suggestion
          should respect this pattern : **wokspace_Name:/regex**.
```

          
The default value of ``exo.webdav.folder.listing.paths.allowed.regex`` is set to empty 

> which means that **the access is allowed to all resources**.

Reviewer: ```suggestion
which means that **the contents of All JCR folders are listed**.
```
